server/controllers/accountController.js

```diff
@@ -21,9 +19,9 @@ export default class AccountController {
     const { accountnumber } = req.params;
     const accountExists = await Account.getAccount(accountnumber);
     if (!accountExists) {
-      return res.status(404).json({
-        status: 404,
-        error: 'Account number not found.',
+      return res.status(400).json({
+        status: 400,
+        error: 'Account does not exist.',
```

Comment:
This should be status 404. Why did you change it to 400?

Reply:
Sorry, I changed it back to 404.